### ISSUE TITLE
ldapadmin - TypeError: _.includes is not a function

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#1270]
privateui does not load anymore on sdi.georchestra.org

```
TypeError: _.includes is not a function
    at Object.b.isSafe (https://cdn.jsdelivr.net/restangular/latest/restangular.min.js:6:534)
    at https://cdn.jsdelivr.net/restangular/latest/restangular.min.js:6:314
    at Array.forEach (native)
    at Function.w.each.w.forEach (https://cdn.jsdelivr.net/underscorejs/1.4.3/underscore-min.js:1:599)
    at c (https://cdn.jsdelivr.net/restangular/latest/restangular.min.js:6:159)
    at f.resource (https://cdn.jsdelivr.net/restangular/latest/restangular.min.js:6:8274)
    at Array.y (https://cdn.jsdelivr.net/restangular/latest/restangular.min.js:6:16768)
    at new UsersCtrl (https://sdi.georchestra.org/ldapadmin/privateui/js/controllers.js:93:15)
    at d (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:28:304)
    at Object.instantiate (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:28:434)
```

**Comment by fvanderbiest** [comment]
root cause = https://github.com/georchestra/georchestra/issues/939 ?

**Comment by fvanderbiest** [comment]
To be fixed from 14.12 on, since this probably affects everyone.

**Comment by fvanderbiest** [comment]
Restangular 1.5.2 pushed to CDN broke the app, see https://github.com/mgonto/restangular/releases
Since it worked great up to now, considering fixing version = 1.5.1
